src/test/ts/core/NpmTagsTest.ts

```diff
@@ -233,5 +248,22 @@ describe('NpmTags', () => {
         });
       });
     });
+
+    describe('arbitrary branches', () => {
+      const specialChar = () => fc.char().filter((char) => !/[\w.]/.test(char));
+      const alphanumeric = () => fc.char().filter((char) => /\w/.test(char));
+
+      it('arbitrary branch names should collapse special characters', () =>
+        fc.assert(fc.asyncProperty(fc.stringOf(alphanumeric(), { minLength: 1 }), fc.stringOf(specialChar(), { minLength: 1 }), (str, special) =>
+          checkSimple(`${str}${special}${str}`, BranchState.Spike, '0.1.5', [ `${str}-${str}` ])
+        ))
+      );
+
+      it('arbitrary dependabot prefixed branch names', () =>
```

Comment:
```suggestion
      it('handles arbitrary dependabot prefixed branch names', () =>
```
Maybe something like this to keep it in sentence form?

Reply:
```suggestion
      it('handles dependabot prefixed branch names', () =>
```